usaspending_api/etl/es_transaction_mapping.json

```diff
@@ -0,0 +1,270 @@
+{
+  "settings": {
+    "index.mapping.ignore_malformed": true,
+    "index.max_result_window": 50000,
+    "index.refresh_interval": -1,
```

Comment:
Is this modified to a better value after the indexing is complete?

Reply:
It is https://github.com/fedspendingtransparency/usaspending-api/blob/6ca492d196d9b416a93afb767f30cbc4520bea23/usaspending_api/etl/es_etl_helpers.py#L286